migration/mule/cache-scope-with-salesforce-contacts/CacheScopeWithSalesforceContacts/src/main/synapse-config/api/ContactsAPI.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<api context="/contacts" name="ContactsAPI" xmlns="http://ws.apache.org/ns/synapse">
+    <resource methods="GET">
+        <inSequence>
+            <cache collector="false" maxMessageSize="10000" timeout="2000">
+                <onCacheHit>
+                    <log level="custom">
+                        <property name="Fetch Contacts" value="Querying from cache..."/>
+                    </log>
+                    <respond/>
+                </onCacheHit>
+                <protocol type="HTTP">
+                    <methods>*</methods>
+                    <headersToExcludeInHash/>
+                    <responseCodes>2[0-9][0-9]</responseCodes>
+                    <enableCacheControl>false</enableCacheControl>
+                    <includeAgeHeader>false</includeAgeHeader>
+                    <hashGenerator>org.wso2.carbon.mediator.cache.digest.HttpRequestHashGenerator</hashGenerator>
+                </protocol>
+                <implementation maxSize="100"/>
+            </cache>
+            <log level="custom">
+                <property name="Fetch Contacts" value="Querying from Salesforce..."/>
+            </log>
+            <salesforcerest.init>
+                <accessToken></accessToken>
+                <apiVersion>v44.0</apiVersion>
+                <hostName>https://login.salesforce.com</hostName>
+                <refreshToken></refreshToken>
+                <apiUrl></apiUrl>
+                <registryPath>connectors/SalesforceRest</registryPath>
+            </salesforcerest.init>
+            <salesforcerest.query>
+                <queryString>SELECT Id, FirstName, LastName, Email, Phone from contact limit 200</queryString>
+            </salesforcerest.query>
+            <datamapper config="gov:datamapper/MapContacts.dmc" inputSchema="gov:datamapper/MapContacts_inputSchema.json" inputType="JSON" outputSchema="gov:datamapper/MapContacts_outputSchema.json" outputType="JSON" xsltStyleSheet="gov:datamapper/MapContacts_xsltStyleSheet.xml"/>
+            <property name="RESPONSE" scope="default" type="STRING" value="true"/>
```

Comment:
Can u explain the need for this property?

Reply:
According to example 2 section in [1], the property is added to consider the message as a response.

[1] https://docs.wso2.com/display/EI620/Cache+Mediator#CacheMediator-Examples

Reply:
@dinuish94 This fact that it needs to be a response in the message context or that we need to mark it as a response should be mentioned on the documentation - we should not let the users grab that fact from the example.

Reply:
Raised: https://github.com/wso2/docs-ei/issues/1509